API-reference.md

```diff
@@ -142,6 +142,7 @@ Parameter | Type | Description
 ---|---|---
 `name` | string | **Required**<br>An event name string.
 `properties?` | dictionary | **Optional**<br>Map of string to any: Additional data used to [filter pages](https://azure.microsoft.com/documentation/articles/app-insights-api-custom-events-metrics/#properties) in the portal. Defaults to empty.
+`measurements?` | dictionary | **Optional**<br>Map of string to any: Additional data (metrics) used to [filter pages](https://azure.microsoft.com/documentation/articles/app-insights-api-custom-events-metrics/#properties) in the portal. Defaults to empty.
```

Comment:
This is actually excluded from the documentation as it's deprecated

Reply:
If you would like to still see it in the documentation please add strikeout around it to indicate that it's deprecated (using double ~~ around the fields.

~~strikeout example~~

Reply:
I think it will help other developers as well to know that this field is deprecated. I updated the API reference accordingly.